act/policy_test.go

```diff
@@ -0,0 +1,26 @@
+package act
+
+import (
+	"context"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestPolicy_Eval(t *testing.T) {
+
+	t.Run("it should eval program correctly", func(t *testing.T) {
+		policy := MustNewPolicy("test", "1 + 1", map[string]any{"log": "enabled"})
+		resp, err := policy.Eval(context.Background(), Input{})
+		require.NoError(t, err)
+		assert.Equal(t, 2, resp)
+	})
+
+	t.Run("it should stop eval if context is timed out", func(t *testing.T) {
+		policy := MustNewPolicy("test", "repeat(\"Hi\", 100000)", map[string]any{"log": "enabled"})
+		ctx, cancel := context.WithTimeout(context.Background(), 10)
+		defer cancel()
+		_, err := policy.Eval(ctx, Input{})
+		require.ErrorIs(t, err, context.DeadlineExceeded)
+	})
+}
```

Comment:
Add another test for an error from policy evaluation.

Reply:
Added a test for a normal runtime error from expr evaluation. 
Also, as adding string and integer was a compile error, I added tests for `MustCompile` as well.